xla_native_functions.yaml

```diff
@@ -1,5 +1,15 @@
 backend: XLA
 cpp_namespace: torch_xla
+per_op_log: XLA_FN_TRACK(3)
+per_argument_log: TF_VLOG(3)
```

Comment:
@ailzhang @JackCaoG I'm actually hoping that I can just completely remove this, but I wanted to confirm with you. Right now, the auto-generated `add_out()` kernel for XLA has the same logging behavior as the original XLA codegen, and looks like this:
```
at::Tensor wrapper_Tensor_add(const at::Tensor &self, const at::Tensor &other,
                              const at::Scalar &alpha) {
  // This is the actual XLA add kernel
  return torch_xla::add(self, other, alpha);
}

}  // anonymous namespace
at::Tensor &wrapper_out_add_out(const at::Tensor &self, const at::Tensor &other,
                                const at::Scalar &alpha, at::Tensor &out) {
  XLA_FN_TRACK(3);
  TF_VLOG(3) << "XLA wrapper_out_add_out :"
             << " self=" << self.toString() << " other=" << other.toString()
             << " out=" << out.toString();

  auto wrapper_out_add_out_tmp = wrapper_Tensor_add(self, other, alpha);
  at::_copy_from_and_resize(wrapper_out_add_out_tmp, out);
  return out;
}
```

That logging information in the generated `add_out` kernel seems unnecessary though, since it ends up calling directly into `torch_xla::add`, which [has its own logging too.](https://github.com/pytorch/xla/blob/280348439b23fd7aef5c57c9b92298f8890bcc98/torch_xla/csrc/aten_xla_type.cpp#L501).

Passing C++ macros directly into yaml so the codegen can plop them into a C++ function is also pretty fragile; what if another backend wants some other more custom logging.

So I think it would be cleaner to kill that code-generated logging and just expect all logging to be done in the functional kernel lowerings, but I wanted to see what your thoughts are!

Reply:
I agree that `auto-generated add_out()` should not have logging and counter if the actual kernel has the logging and counter already.